docs/rule/no-route-action.md

```diff
@@ -0,0 +1,51 @@
+# no-route-action
+
+This rule disallows the usage of `route-action`.
+
+[ember-route-action-helper](https://github.com/DockYard/ember-route-action-helper) is a popular addon that it's only purpose was to avoid creating controllers, something we no longer have a desire to do given the current state of the framework.
```

Comment:
```suggestion
[ember-route-action-helper](https://github.com/DockYard/ember-route-action-helper) was a popular addon used to avoid creating controllers just to add actions to routes. Given the changes in Ember since ember-route-action-helper was useful, controllers are now a useful pattern we want to encourage and discourage the use of route-action.
```